lib/rack/request_police/middleware.rb

```diff
@@ -25,6 +25,25 @@ def call(env)
             if %w(POST PATCH DELETE).include?(env['REQUEST_METHOD'])
               request_params.merge!('data' => utf8_input(env))
             end
+
+            headers = Rack::RequestPolice.headers.inject({}) do |result, header_hash|
+              header_name = header_hash.fetch(:original_header_name)
+              transformation = header_hash.fetch(:transformation)
+
+              fallback_value = header_hash.fetch(:fallback_value)
+              storage_name = header_hash.fetch(:storage_name)
+
+              header_value = env[header_name]
+              # apply transformations to header_value, fail silently on any exception
+              header_value = transformation.call(header_value) rescue header_value
```

Comment:
Well, me neither

But..

If user specifies wrong transaction, for example
```ruby
{ |h| h.some_not_existing_method }
```
Seems reasonable to me, to rescue to original `header_value` if that transformation fails instead of returning 500 ;)